src/utils/helpers/browser/browser.js

```diff
@@ -60,6 +60,16 @@ const Browser = {
   },
 
   /**
+   * Focues and sets cursor of input filed but does not select text
+   *
+   * @method setFocus
+   */
+  setFocus: (ref) => {
+    let node = ReactDOM.findDOMNode(ref._input);
```

Comment:
This definitely needs to be `setInputFocus`

But I also like @SeanArmstrong idea, focus is a DOM element thing, not just an input thing.